foundry.toml

```diff
@@ -0,0 +1,15 @@
+[default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+solc-version = "0.8.7"
+optimizer = true
+optimizer_runs = 1000000
+verbosity = 1
+fuzz_runs = 5 
+remappings = [
+    'ds-test/=lib/ds-test/src/',
```

Comment:
`forge-std` has a dependency on `ds-test`: https://github.com/brockelmore/forge-std/tree/master/lib, can we reuse it with something like `'ds-test/=lib/forge-std/lib/ds-test/src'`? Having two version in a project is a ticking bomb. I also wouldn't be surprised if forge created its own version, the original has some aspect to improve and it may need to start using the new cheatcodes.

Reply:
Good point.